controllers/create_stream.py

```diff
@@ -30,13 +30,12 @@ def post(self):
         subs_emails = self.request.get_all('subscriber-emails')
         stream_tags = self.request.get_all('stream-tags')
         cover_url = self.request.get('stream-cover-url')
-        owner_email = self.request.get('stream-owner')
         stream_data = { 'name': stream_name,
                         'emails': subs_emails, 
                         'email_message': invite_message, 
                         'tags': stream_tags,
                         'cover_url': cover_url, 
-                        'owner': owner_email }
+                        'owner': user.user_id() }
```

Comment:
Ok with this change. I'll need to create a bunch of stuff again so that it's tied to my id and not my email but I forgive you since I was responsible for the use of email in the first place :)